src/main/java/edu/escuelaing/ieti/matchmaking/exception/handler/ApiError.java

```diff
@@ -0,0 +1,55 @@
+package edu.escuelaing.ieti.matchmaking.exception.handler;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import org.springframework.http.HttpStatus;
+
+import java.time.LocalDateTime;
+
+public class ApiError {
+
+    private HttpStatus status;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
```

Comment:
You can annotate every field this way or you can customize your marshaller to handle every Date the same way in your whole API. Try something like this in your configuration class:

`@Bean public Jackson2ObjectMapperBuilder objectMapperBuilder() { Jackson2ObjectMapperBuilder builder = new Jackson2ObjectMapperBuilder(); builder. simpleDateFormat("yyyy-MM-dd'T'HH:mm:ss"); return builder; }`